main.go

```diff
@@ -39,10 +39,11 @@ const (
 
 // Config - configuration for cmd-exclude-prefixes-k8s
 type Config struct {
-	ExcludedPrefixes   []string `desc:"List of excluded prefixes" split_words:"true"`
-	ConfigMapNamespace string   `default:"default" desc:"Namespace of kubernetes config map" split_words:"true"`
-	ConfigMapName      string   `default:"nsm-config" desc:"Name of kubernetes config map" split_words:"true"`
-	PrefixesFilePath   string   `default:"/var/lib/networkservicemesh/config/excluded_prefixes.yaml" desc:"Excluded prefixes file absolute path" split_words:"true"`
+	ExcludedPrefixes      []string `desc:"List of excluded prefixes" split_words:"true"`
+	ConfigMapNamespace    string   `default:"default" desc:"Namespace of user config map" split_words:"true"`
+	ConfigMapName         string   `default:"excluded-prefixes-config" desc:"Name of user config map" split_words:"true"`
+	NSMConfigMapNamespace string   `default:"default" desc:"Namespace of nsm config map" split_words:"true"`
```

Comment:
Can exclude prefix service find it's a namespace? In this case, we can remove this parameter.

Reply:
Now we get current namespace from mounted serviceaccount